Resources/Prototypes/_NF/Research/arsenal.yml

```diff
@@ -17,6 +58,39 @@
   - AdvancedRiotControl
   - HardsuitsAdvanced
 
+- type: technology
+  id: ArsenalStyle
+  name: research-technology-arsenal-style
+  icon:
+    sprite: Objects/Weapons/Melee/katana.rsi
+    state: icon
+  discipline: Arsenal
+  tier: 2
+  cost: 15000
```

Comment:
I might be missing something, why is this so expensive?  Elite punk armor?  Just a lot of things?

Reply:
Mostly because research points are infinite.